oeps/oep-0002-bp-repo-metadata.rst

```diff
@@ -29,74 +30,119 @@ Proposes a standard format for repository metadata for Open edX projects.
-       in reporting tools. If an OEP isn't listed in the ``oeps`` dictionary, then
-       it is assumed to be ``False``, unless the reporting tool can auto-detect
-       accordance.
+    -  If the ``state`` is ``True`` or ``applicable`` is ``False`` the ``reason`` can optionally provide more information. The ``reason`` value will be displayed in reporting tools. If an OEP isn't listed in the ``oeps`` dictionary, then it is assumed to be ``False``, unless the reporting tool can auto-detect accordance.
 
 ``track-pulls``: obsolete
     This key is obsolete, please remove if found.
 
-``openedx-release``: defined by :doc:`OEP-10 <oep-0010-proc-openedx-releases>`.
-    Define this key if your repo is an application or IDA that is part of
-    Open edX releases.  Omit this key if your repo is a library, or is not part
-    of Open edX releases.  See :doc:`OEP-10 <oep-0010-proc-openedx-releases>`
-    for details.
+``openedx-release``: defined by :doc:`OEP-10 <oep-0010-proc-openedx-releases>` (optional)
+    Define this key if your repo is an application or IDA that is part of Open edX releases.  Omit this key if your repo is a library, or is not part of Open edX releases.  See :doc:`OEP-10 <oep-0010-proc-openedx-releases>` for details.
 
 ``archived``: boolean (optional)
-    If ``True``, this specifies that this repository is archived and no longer
-    maintained by edX.
+    If ``True``, this specifies that this repository is archived and no longer maintained by edX.
+
+Tags
+----
+
+The ``tags`` structure is very open ended but there are a few tags that are currently in wide use and are provided here so that they can be used consistently.
+
+``webservice``:
+    The repository is the root of a webservice that serves HTTP content.
+
+    Code runs in production web servers and has publicly accessible views/APIs for end-users.
+
+    eg. credentials
+
+``frontend-app``:
+    The repository primarily contains code for a frontend application.
+
+    Code runs in production devices (such as web browsers and mobile devices) and is publicly accessible by end-users.
+
+    eg. frontend-app-profile, edx-app-ios
+
+``framework``:
+    The repository contains shared functionality that form a common framework used by multiple services or apps in the platform.
 
+    Code runs in production, behind the scenes, (intended to be) globally used throughout the platform.
+
+    eg. edx-drf-extensions, edx-ace, frontend-platform, paragon
+
+``backend-service``:
+    A service that is run as part of the platform but is not directly accessed by end-users.
+
+    Code runs in production backend servers, typically within the firewall, and provides APIs for access by public-facing ``webservices``.
+
+    eg. xqueue-watcher, ecommerce-worker
+
+``backend-tooling``:
+    The repository contains scripts or configuration used in conjunction with backend services or in support of openedx.
+
+    Code does *not* run in production.
```

Comment:
The `configuration` and `terraform` reepo feel like they would get this tag if it were not for this line.  Both of those repos have code that "runs in" production in that they build/maintain production.

Is there a different tag you would give those tools or should we just remove this line from the description?

Reply:
In our draft attempt of `Ownership Assignment`, we listed `terraform` as a `framework`. 

I see potential value in us knowing which of our repos are never run in production. It may help with resourcing and prioritization efforts for upgrades/etc.

Reply:
Ah, cool, that sounds good.